src/main/java/com/autotune/analyzer/deployment/KruizeDeployment.java

```diff
@@ -161,7 +161,7 @@ public static void processKruizeObject(KruizeObject kruizeObject) {
                 List<KruizeObject> kruizeObjectList = new ArrayList<>();
                 kruizeObjectList.add(kruizeObject);
                 ExperimentInitiator experimentInitiator = new ExperimentInitiator();
-                ActivityResultData activityResultData = experimentInitiator.validateAndAdd(autotuneObjectMap, kruizeObjectList);
+                ActivityResultData activityResultData = experimentInitiator.validateAndAddNewExperiments(autotuneObjectMap, kruizeObjectList);
                 if (!activityResultData.isSuccess()) {
```

Comment:
Can we unify the logging in both of the cases of starting and experiment (k8s native and API). Right now it appears the logging happens in one and not in the other.